var/spack/repos/builtin/packages/py-datalad/package.py

```diff
@@ -110,3 +110,10 @@ class PyDatalad(PythonPackage):
         depends_on('py-pillow', type=('build', 'run'))
         # duecredit
         depends_on('py-duecredit', type=('build', 'run'))
+
+    depends_on('py-nose', type=('test'))
+    install_time_test_callbacks = ['test', 'installtest']
+
+    def installtest(self):
+        datalad = Executable('{0}/datalad'.format(self.prefix.bin))
```

Comment:
```suggestion
        datalad = Executable(self.prefix.bin.datalad)
```
You could also add a `commands` attribute like we have for Python and call that instead, but that's more complicated.

Reply:
Thanks!